src/eval.rs

```diff
@@ -39,9 +39,9 @@ pub fn create_ecx<'mir, 'tcx: 'mir>(
         Evaluator::new(config.communicate),
         MemoryExtra::new(StdRng::seed_from_u64(config.seed.unwrap_or(0)), config.validate),
     );
-
-    ShimsEnvVars::init(config.communicate, &mut ecx, &tcx);
-
+    // Complete initialization.
+    EnvVars::init(&mut ecx, &tcx, config.communicate);
+    // Setup first stack-frame
```

Comment:
Please leave an empty line above.

Code is a lot like text: paragraph help to structure it.

Reply:
You resolved this, I can't see the empty line (above "setup first stack frame")?

Reply:
Sorry apparently I missed it. It is done now